src/Calendar/Calendar.test.js

```diff
@@ -226,7 +226,9 @@ describe('<Calendar />', () => {
 
     test('click previous button', () => {
         let wrapper = mount(defaultCalendar);
-        const currentDateDisplayed = Object.assign(new Date(), wrapper.state('currentDateDisplayed'));
+
+        let initialDate = new Date('3/28/2019');
+        wrapper.setState({ currentDateDisplayed: initialDate });
```

Comment:
There is a datetime testing lesson to be learned from this here...

Reply:
If only we learned that in #329 and #342 😸

Reply:
and apparently, #493 as well.